src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -637,10 +603,30 @@ robocopy /S /NP %EXECUTION_DIR%int\%(Identity)\ %EXECUTION_DIR%native\
 
   <Target Name="SetupTestProperties" DependsOnTargets="CheckTestPlatforms;CheckTestCategories" />
 
+  <PropertyGroup Condition="$(IsTestProject) == 'true' AND '$(OmitLaunchSettings)' != 'true'">
+    <LaunchSettingsOutputFileFullPath>$(AppDesignerFolder)/launchSettings.json</LaunchSettingsOutputFileFullPath>
+    <LaunchSettingsInputFileFullPath>$(MSBuildThisFileDirectory)launchSettings.json</LaunchSettingsInputFileFullPath>
+
+    <PrepareForRunDependsOn>
+      GenerateLaunchSettingsFile;
+      CopySupplementalTestData;
+      $(PrepareForRunDependsOn);
+    </PrepareForRunDependsOn>
+  </PropertyGroup>
+
+  <Target Name="GenerateLaunchSettingsFile"
+          Condition="'$(OmitLaunchSettings)' != 'true'"
+          Inputs="$(LaunchSettingsInputFileFullPath)"
+          Outputs="$(LaunchSettingsOutputFileFullPath)">
+
+    <Copy SourceFiles="$(LaunchSettingsInputFileFullPath)"
+          DestinationFolder="$(AppDesignerfolder)" />
```

Comment:
Maybe instead use DestinationFile with `$( LaunchSettingsOutputFileFullPath)` then it matches inputs /outputs and if you change the destination from AppDesignerFolder you don't need to change it here too

Reply:
If the file exists does it overwrite without complaint?

Reply:
thanks that was an oversight, changed it to `DestinationFiles`. Yes overwrite is the default for the copy task.